CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set(MANIFOLD_FLAGS -O3)
 
 if(EMSCRIPTEN)
   message("Building for Emscripten")
-  set(MANIFOLD_FLAGS -fexceptions)
+  set(MANIFOLD_FLAGS -fexceptions -pthread)
```

Comment:
According to [emscripten](https://emscripten.org/docs/porting/pthreads.html), this is all that's needed, but I'm getting the same compile error from Thrust.